source/installation/install-on-a-single-machine/install-on-a-single-machine.txt

```diff
@@ -10,7 +12,52 @@ Installation on a Single Machine
    :depth: 1
    :class: singlecol
 
-TBD.
+You can install Relational Migrator on a single machine. The single machine 
+installation provides a simplified installation process by using default settings.
+This installation method is a "local" install because Relational Migrator 
+cannot be externally accessed and is bound to a single IP and port. 
+
+Use Cases
+---------
+
+Installing Migrator on a single machine is ideal for small developmental projects 
+that do not require not external access. 
+
+Behavior
+--------
+
+The JDBC drivers must be downloaded separately, and placed into a dedicated 
+directory after Relational Migrator is installed. This is required to connect to 
+your relational database. The JDBC driver directory is operating system specific 
+and included in the installation instructions below.
+
+If you are migrating from a MySQL database:
+
+- Download the `latest 8.0.x version, Platform Independent Version <https://dev.mysql.com/downloads/connector/j/>`_.
+
+- Unzip the downloaded file and only copy the ``.jar`` file into the target 
```

Comment:
(typo?)
Unzip the downloaded file and copy the ... 

Should we omit only?

Reply:
Done.